wallet/rpcrawtransaction.cpp

```diff
@@ -48,13 +48,24 @@ void ScriptPubKeyToJSON(const CScript& scriptPubKey, Object& out, bool fIncludeH
 
 void TxToJSON(const CTransaction& tx, const uint256 hashBlock, Object& entry)
 {
+    auto pair = std::make_pair(FetchTxFromDisk(tx.GetHash()), NTP1Transaction());
+    bool isNTP1 = IsTxNTP1(&tx);
+    if (isNTP1) {
+        CTxDB                  txdb("r");
+        FetchNTP1TxFromDisk(pair, txdb, false);
+        if (pair.second.isNull()) {
+        	isNTP1 = false;
```

Comment:
It's probably not the worst idea to throw an exception here. If a transaction is NTP1 in OP_RETURN and it's not NTP1 in the db, then something is wrong.

Reply:
I actually hit this in early testing and it crashed the node. The problem is that there are transactions that start with 4e5401|03 but for whatever various reason have been deemed invalid by the consensus rules and therefore do not exist in the db. I think we want to handle these cases gracefully and return `tokens: []` in the output instead of throwing an exception, don't you?

Reply:
I see what you mean. That's fair.